projects/table-builder/src/lib/components/header-menu/header-menu.component.html

```diff
@@ -12,9 +12,9 @@
       <input type="hidden" name="key"  [ngModel]="metaData.key" />
       <input type="hidden" name="fieldType"  [ngModel]="metaData.fieldType" />
 
-      <ng-container *ngIf="metaData.additional?.FilterOptions?.FilterableValues?.length > 0; else defaultFilter">
+      <ng-container *ngIf="valueList; else defaultFilter">
         <input type='hidden' name='filterValue' [ngModel]='myFilterValue' />

```

Comment:
I guess the myFilterValue is being picked up somewhere?

Reply:
correct